INSTALL-DOCKER.md

```diff
@@ -57,7 +57,7 @@ You can now access these from your browser:
 - Hit `Ctrl+C` to turn off containers.
 - Type `docker-compose up` to start them up again.
 - You might want to run containers as daemon: `docker-compose up -d` and then read the logs only from "trustroots" container: `docker-compose logs trustroots`. You can use a helper script: `./scripts/docker/up.sh`
-- If you see a lot of `Run migrate DB script on update` coming from Nginx container, it means NodeJS isn't up yet or it has stopped due error. You should see "Trustroots server is up and running now." once Trustroots is really running.
+- If you see a lot of `Run migrate DB script on update` coming from Nginx container, it means NodeJS isn't up yet or it has stopped due tp errors. You should see "Trustroots server is up and running now." once Trustroots is really running.
```

Comment:
Typo "tp"

Reply:
Man, I made a typo in my typo PR :sweat_smile: Will fix that, good catch!